index.html

```diff
@@ -0,0 +1,931 @@
+                            </h3>
+                        </a>
+                        <span>5h Ago</span>
+                    </div>
+
+
+                    <figure class="figure-size">
+                        <img src="./assets/img/spaceX.PNG" alt="SpaceX plan image" class="im-geo">
+                        <div>
+                            <div class="img-footer-author">Peter Komka/EPA, via Shutterstock</div>
+                        </div>
+                        <a href="#">
+                            <h3>paceX Plans Sunshades to Save Night Skies From Starlink Satellites</h3>
+                        </a>
+                        <span>May 6</span>
+                    </figure>
+
+
+                    <figure class="figure-size">
+                        <img src="./assets/img/deep_in_the_cosmic.png" alt="Deep in Cosmic image" class="im-geo">
+                        <div>
+                            <div class="img-footer-author">NASA, ESA and D. Lin</div>
+                        </div>
+                        <a href="#">
+                            <h3>Deep in the Cosmic Forest, a Black Hole Goldilocks Might Like</h3>
+                        </a>
+                        <span>May 11</span>
+                    </figure>
+
+                    <figure class="figure-size">
+                        <img src="./assets/img/risky-bussiness.PNG" alt="risky-bussiness image" class="im-geo">
+                        <div>
+                            <div class="img-footer-author">NASA</div>
+                        </div>
+                        <a href="#">
+                            <h3>Risky Business? NASA and Tom Cruise Talk Movie Shoot in Space</h3>
+                        </a>
+                        <span>May 5</span>
+                    </figure>
+
+
+                    <figure class="figure-size">
+                        <img src="./assets/img/NASA-picks.PNG" alt="NASA" class="im-geo">
+                        <div>
+                            <div class="img-footer-author">NASA</div>
+                        </div>
+                        <a href="#">
+                            <h3>This NASA Mission May Cause an Artificial Meteor Shower</h3>
+                        </a>
+                        <span>May 12</span>
+                    </figure>
+
+                    <figure class="figure-size">
+                        <img src="./assets/img/there-is-something.PNG" alt="Sun image" class="im-geo">
+                        <div>
+                            <div class="img-footer-author">NASA</div>
+                        </div>
+                        <a href="#">
+                            <h3>There’s Something Special About the Sun: It’s a Bit Boring</h3>
+                        </a>
+                        <span>April 30</span>
+                    </figure>
+                </div>
+
+
+
+                <div class="editors-pick">
```

Comment:
Must be an [`<aside>` tag](https://www.w3schools.com/tags/tag_aside.asp) for `Editor's picks` section on the rightmost side. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/aside_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))